doc/quick-reference-guide.md

```diff
@@ -0,0 +1,185 @@
++ `defcfg`: configuration rules (input and output devices, etc).
+
++ `input` and `output`:
+	- For Linux:
+       ```
+       input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+       output (uinput-sink "My KMonad output")
+       ```
+	- For Windows:
+       ```
+       input  (low-level-hook)
+       output (send-event-sink)
+       ```
+	- For MacOS:
+		```
+		input  (iokit-name "my-keyboard-product-string")
+		output (kext)
+		```
++ `fallthrough` (boolean, defaults to `false`): allow all keys, even if not defined
+in defsrc.
+
++ `allow-cmd` (boolean, defaults to `false`): allow commands to be run as keys (very
+dangerous).
+
++ `cmp-seq` (key, defaults to `RightAlt`): compose key for Unicode input (Linux X11
+specific).
+
+	+ `cmp-seq-delay` (number): delay between each pressed key in a compose-key
+sequence
+```
+(defcfg
+  ;; For Linux
+  input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+  output (uinput-sink "My KMonad output" "/run/current-system/sw/bin/sleep 1 && /run/current-system/sw/bin/setxkbmap -option compose:ralt")
+
+  cmp-seq ralt
+  cmp-seq-delay 5
+
+  fallthrough true
+
+  allow-cmd true
+)
+```
+
++ `defsrc`: define layer of input keys
```

Comment:
Might be worth mentioning that this should reflect the physical keyboard and is _not_ going to be used as any layer

Reply:
Resolved in 3c0f1d7